test/src/main/java/org/apache/accumulo/test/FileArchiveIT.java

```diff
@@ -192,81 +194,88 @@ public void testUnusuedFilesAndDeletedTable() throws Exception {
-    conn.tableOperations().offline(tableName, true);
+      // Offline the table so we can be sure there is a single file
+      conn.tableOperations().offline(tableName, true);
 
-    // See that the file in metadata currently is
-    s = conn.createScanner(MetadataTable.NAME, Authorizations.EMPTY);
-    s.setRange(MetadataSchema.TabletsSection.getRange(tableId));
-    s.fetchColumnFamily(MetadataSchema.TabletsSection.DataFileColumnFamily.NAME);
+      // See that the file in metadata currently is
+      s = conn.createScanner(MetadataTable.NAME, Authorizations.EMPTY);
+      s.setRange(MetadataSchema.TabletsSection.getRange(tableId));
+      s.fetchColumnFamily(MetadataSchema.TabletsSection.DataFileColumnFamily.NAME);
 
-    entry = Iterables.getOnlyElement(s);
-    final String finalFile = entry.getKey().getColumnQualifier().toString();
-    final Path finalPath = new Path(finalFile);
+      entry = Iterables.getOnlyElement(s);
+      final String finalFile = entry.getKey().getColumnQualifier().toString();
+      final Path finalPath = new Path(finalFile);
 
-    conn.tableOperations().delete(tableName);
+      conn.tableOperations().delete(tableName);
 
-    log.info("File for table: {}", finalPath);
+      log.info("File for table: {}", finalPath);
 
-    i = 0;
-    while (fs.exists(finalPath)) {
-      i++;
-      Thread.sleep(1000);
-      if (0 == i % 10) {
-        log.info("Waited {} iterations, file still exists", i);
+      i = 0;
+      while (fs.exists(finalPath)) {
+        i++;
+        Thread.sleep(1000);
+        if (0 == i % 10) {
+          log.info("Waited {} iterations, file still exists", i);
+        }
       }
-    }
 
-    log.info("File was removed");
+      log.info("File was removed");
 
-    String finalFilePath = finalPath.toUri().getPath().substring(getCluster().getConfig().getAccumuloDir().toString().length());
+      String finalFilePath = finalPath.toUri().getPath().substring(getCluster().getConfig().getAccumuloDir().toString().length());
 
-    log.info("File relative to accumulo dir: {}", finalFilePath);
+      log.info("File relative to accumulo dir: {}", finalFilePath);
 
-    Assert.assertTrue("File archive directory didn't exist", fs.exists(fileArchiveDir));
+      Assert.assertTrue("File archive directory didn't exist", fs.exists(fileArchiveDir));
 
-    // Remove the leading '/' to make sure Path treats the 2nd arg as a child.
-    Path finalArchivedFile = new Path(fileArchiveDir, finalFilePath.substring(1));
+      // Remove the leading '/' to make sure Path treats the 2nd arg as a child.
+      Path finalArchivedFile = new Path(fileArchiveDir, finalFilePath.substring(1));
 
-    Assert.assertTrue("File doesn't exists in archive directory: " + finalArchivedFile, fs.exists(finalArchivedFile));
+      s.close();
+      Assert.assertTrue("File doesn't exists in archive directory: " + finalArchivedFile, fs.exists(finalArchivedFile));
+    } finally {
+      if (s != null) {
```

Comment:
This finally block indicates another possible opportunity for try-with-resources.